.github/workflows/check.yaml

```
@@ -191,7 +191,7 @@ jobs:
           # Create the path that is expected to have a coverage.xml for tics
           mkdir -p tests/report/
 
-          coverage_files=$(echo ./artifacts/.coverage-*)
+          coverage_files=(./artifacts/.coverage-*)
```

The parens must be removed now here, otherwise it's treated as a bash array, and the logic further on won't work the same any more.

```suggestion
          coverage_files=./artifacts/.coverage-*
```

---

I use zsh and apparently bash and zsh interpret this differently. I've checked on bash and your suggestion works, but breaks on zsh.

Too much confusion for a simple thing.  Leaving echo as it was initially wasn't a big deal, but in any case it's as you suggested.